docs-gen/includes/quickstart/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
+name = "quickstart"
+version = "0.1.0"
```

Comment:
It would be great if this could be set dynamically (like in the compiler) but this doesn't need to happen in this PR